Documentation/git-commit.txt

```diff
@@ -177,7 +177,7 @@ The `-m` option is mutually exclusive with `-c`, `-C`, and `-F`.
 	sole parent commit is a mistake, and the command prevents you
 	from making such a commit.  This option bypasses the safety, and
 	is primarily for use by foreign SCM interface scripts. See the 
-	`commit.allowEmpty` configuration variable in linkgit:git-config[1].
+	`commit.allowEmpty` in linkgit:git-config[1].
 
 --allow-empty-message::
        Like --allow-empty this command is primarily for use by foreign
```

Comment:
Maybe a regression test would make sense, to verify that this feature works as intended?

Reply:
please review the tests added. :)

Reply:
Thanks, I added a comment there.